zap/src/main/java/org/zaproxy/zap/extension/search/SearchThread.java

```diff
@@ -279,6 +279,14 @@ private void search() {
                             }
                         }
                     }
+                    if (Type.Note.equals(reqType) && !pcc.allMatchesProcessed()) {
+                        String note = message.getHistoryRef().getHttpMessage().getNote();
```

Comment:
Why not get the note directly?

Reply:
Probably just lack of familiarity with the code base. Kinda seems like something I woulda done too.

@SkypLabs you can `message.getNote()` without using the history ref. (That's what you meant right @thc202?)
https://javadoc.io/static/org.zaproxy/zap/2.15.0/org/parosproxy/paros/network/HttpMessage.html#getNote()

Reply:
Indeed, I didn't realise that `message` and the return value of `getHttpMessage()` were of the same type.

I was actually surprised to be able to get the tags and the alerts from `getHistoryRef()` (e.g. `message.getHistoryRef().getTags()`) but not the note.

Reply:
Fixed in 1efbf2860.